video_frame_metadata_registry.src.html

```diff
@@ -61,6 +61,18 @@
     <td>segments</td>
     <td>[Human face segmentation](https://w3c.github.io/mediacapture-extensions/#human-face-segmentation)</td>
   </tr>
+  <tr>
+    <td>captureTime</td>
+    <td>[Capture time](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-capturetime)</td>
+  </tr>
+  <tr>
+    <td>receiveTime</td>
+    <td>[Receive time](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-receivetime)</td>
+  </tr>
+  <tr>
+    <td>rtpTimestamp</td>
+    <td>[RTP timestamp](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-rtptimestamp)</td>
```

Comment:
The RVFC text is ok here but I'd still probaby copy it over rather than referencing it.

Reply:
I planned to define what a "rtp timestamp" is over at webrtc-extensions and simply define here that it's present for "remote webrtc tracks" together with a reference.